packages/frontend/web-ui/src/game/models/FormValidationNewGameResult.ts

```diff
@@ -0,0 +1,3 @@
+export interface FormValidationNewGameResult {
```

Comment:
Let's assume this represents validation errors for the sake of the `Either` pattern usage.
```suggestion
export interface FormNewGameValidationErrorResult {
```

Reply:
Changed!